Ryujinx.HLE/HOS/Services/Mii/Types/StoreData.cs

```diff
@@ -78,7 +76,7 @@ private ushort CalculateDeviceCrc()
 
```

Comment:
These should be annotated with `UnscopedRef` because `this` escapes the scope of the method.
```suggestion
        [UnscopedRef]
```